plugin/src/server/sc/plugin2021/GamePlugin.kt

```diff
@@ -31,9 +31,14 @@ class GamePlugin: IGamePlugin {
                     Color::class.java, Team::class.java)
         
         fun registerXStream() {
-            if (loaded.compareAndSet(false, true)) {
+            if (loaded.compareAndSet(-1, 0)) {
```

Comment:
Please add a comment explaining that this is required for thread safety.

Reply:
Yes, I am a bit unsure now actually. This issue is _only_ relevant for the parallelized tests, in normal usage there is no thread safety needed. So this seems a bit over the top...